src/DemoData.hs

```diff
@@ -156,6 +187,9 @@ universe rnd = void $ do
 
 -- * Helpers
 
+-- This 'Monad m => m a' is strange, why not simply 'a' instead?
+-- Second, is this actually safe to use the same generator over and over or
+-- does it need to be threaded?
```

Comment:
there was a version, where I used split, but the code was noisy, when we use this module to generate better test data, not just demo data, we can use split and safer approach.